R/target_market_share.R

```diff
@@ -450,7 +450,12 @@ separate_metric_from_name <- function(data) {
       name = sub("(production)_", "\\1-", .data$name),
       name = sub("(technology_share)_", "\\1-", .data$name)
     ) %>%
-    tidyr::separate(.data$name, into = c("name", "metric"), sep = "-")
+    tidyr::separate(
+      .data$name,
+      into = c("name", "metric"),
+      sep = "-",
+      extra = "merge"
```

Comment:
shockingly simple fix :)

Reply:
My favourite kind!

I am going to try to address your knit comment while I'm here also.

Reply:
Ah actually no I'm not. That new function is still experimental, I will wait until it is stable.

Reply:
New issue #471